Add interference solution for Z-Wave JS troubleshooting (#19841)
home-assistant/home-assistant.github.io: source/_integrations/zwave_js.markdown

When trying to determine why something isn't working as you expect, or when reporting an issue with the integration, it is helpful to know what Z-Wave JS sees as the current state of your Z-Wave network. To get a dump of your current network state, follow the menu:

**Configuration** -> **Integrations** -> **Z-Wave JS** -> **Configure** -> **Download a dump of your network to help diagnose issues**

### Interference issues

> A lot of users will have issues with interference if the USB stick is directly connected to the machine (proximity), if you are having issues try to use a short USB 2.0 A male to female extension cord

Reviewer: ```suggestion
Many users have reported issues with interference when the USB stick was directly connected to the machine (proximity). If you are having issues try to use a short USB 2.0 A male to female extension cord.
```
